docs/reference/commandline/cli.md

```diff
@@ -45,6 +45,7 @@ by the `docker` command line:
   unsuitable for Docker.
 * `DOCKER_RAMDISK` If set this will disable 'pivot_root'.
 * `DOCKER_TLS_VERIFY` When set Docker uses TLS and verifies the remote.
+* `DOCKER_NOTARY` When set Docker uses notary to sign and verify images.
```

Comment:
Notary is an implementation detail of the trust feature of Docker. Do we want to make it appear all the way up to the UX? Isn't making the name appear here contradictory to the idea of having well spec'ed ingredients that we can substitute with other conforming implementations?